### ISSUE TITLE
Clicks and actions

### ISSUE THREAD

**Issue by TiberiusB** [valnet/valuenetwork#414]
We noticed that if we click many ties on a Save button the action might be duplicated. 
For example, when we log time, and create a work event, if the system is slow, people click more than once on the button and the result is multiple work events created. 

Possible solutions: have a sign to tell the user that the action has been recorded and disable the UI until the page refreshes.

**Comment by bhaugen** [comment]
Good catch, thanks, we fixed a similar problem in the new equipment logging
app and will do so as comprehensively as possible.
